pkg/chart/loader/archive_test.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package loader_test
```

Comment:
can you change this to `package loader` as per standard Go convention?

See also https://github.com/helm/helm/blob/af59e32654f048d62b053afb0860a3bec77444fb/pkg/chart/loader/load_test.go#L17

Reply:
@bacongobbler Sure... looks like I'm outdated on my conventions, as I thought that was the preferred way to enforce that tests to use the exported interface, but a quick search isn't finding canonical guidance.

Reply:
```
$ go help test
...
Test files that declare a package with the suffix "_test" will be compiled as a
separate package, and then linked and run with the main test binary.
...
```

which isn't exactly a ringing "you should do it this way"...